model/src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala

```diff
@@ -509,6 +528,32 @@ object WorkflowStatuses {
   case object Unknown extends WorkflowStatus
 }
 
+object SubmissionRetryStatuses {
+  sealed trait RetryStatus extends RawlsEnumeration[RetryStatus] {
+    def filterWorkflows(workflows: Seq[Workflow]) = {
+      this match {
+        case RetryFailed => workflows.filter(wf => wf.status.equals(Failed))
+        case RetryAborted => workflows.filter(wf => wf.status.equals(Aborted))
+        case RetryBoth => workflows.filter(wf => wf.status.equals(Failed) || wf.status.equals(Aborted))
```

Comment:
nit: change this to `RetryFailedAndAborted` or similar - "both" is not clear without knowing that the other options are failed and aborted